src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,28 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
+    public String getUniqueCharacters(String fileName) throws IOException {
+        StringBuilder resultBuilder = new StringBuilder();
+        TreeSet<Character> resultSet = new TreeSet<>();
+        try (BufferedReader fileReader = new BufferedReader(new FileReader(fileName))) {
+            while (fileReader.ready()) {
+                char[] letters = fileReader.readLine().replaceAll("[^a-zA-Z]", "")
+                        .toLowerCase().toCharArray();
+                for (int i = 0; i < letters.length; i++) {
+                    resultSet.add(letters[i]);
+                }
+            }
+            for (Character symbol : resultSet) {
```

Comment:
for (Character symbol : resultSet) {
                resultBuilder.append(symbol);
 if (resultBuilder.length() == 5) {
               brake;
            }
            }
            return resultBuilder.toString();